drift_dev/.gitignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
nit: I think it would be nicer to add `*/build/` to the top-level gitignore instead. That way, we get the same behavior in all top-level packages in this repository. Currently, if you ran `flutter test` in `drift/` you'd probably run into the same issue.